src/content/contributing/adding-layer-2/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Adding Layer 2's
+description: The policy we use when adding layer 2's to ethereum.org
+lang: en
+sidebar: true
+---
+
+# Adding layer 2's {#adding-layer-2}
+
+Anyone is free to suggest new layer 2's on ethereum.org.
+
+We currently list them on:
+
+- [ethereum.org/layer-2](/layer-2/)
+
+This page is a beginner/intermediate level page on layer 2's, and on this page a user is able to see some prominent layer 2 projects that they can easily get onboarded on.
+
+Because of this context, we need some specific information when you suggest a layer 2.
+
+## What we need {#what-we-need}
+
+Some information we will need beyond standard information of the layer 2 are:
+
+- [L2Beat page](https://l2beat.com) - in order to be considered we need this project to be featured on L2beat. This resource provides a great risk assessment of layer 2 projects. If the project is not featured on L2Beat, they will not be considered for ethereum.org.
+- Is the project open source?
+- What kind of layer 2 is it? Optimistic? ZK? Validium?
+- Bridges - how are users able to onboard to the layer 2?
+- Exchange support - are users able to deposit directly from an exchange?
+- How long has the layer 2 been live?
+- Links to dapps in the layer 2 ecosystem - we want to be able to provide information on what users can expect to be able to do on these layer 2's.
+- Token contract lists - since assets will have a new address on layer 2, we would like to be able to help users onboard with trusted resources for token contracts on the layer 2.
+- Is the project a fork?
+- Is the project audited?
+
+We need this information so that we can provide the best onboarding experience for users learning about layer 2's.
+
+## Some additional things we may look for {#some-additional-things-we-may-look-for}
+
+TVL of the layer 2. We want to see sustained adoption of the network, so if it is only being used short term for things like yield farming and then liquidity leaves, we may take this into consideration when choosing to accept a project or not.
```

Comment:
I don't see a reason to separate this into a separate section. I think this can be part of the larger list of e.g. "How we decide".